src/Ubench.php

```diff
@@ -166,4 +166,14 @@ public static function readableElapsedTime($microtime, $format = null, $round =
 
         return sprintf($format, $time, $unit);
     }
+
+    public function wasEnd()
```

Comment:
sorry I missed that the first time but i think the correct name should be: hasEnded and hasStarted

Can you make this change?